test/fluree/db/general/connection_test.cljc

```diff
@@ -0,0 +1,258 @@
+(ns fluree.db.general.connection-test
```

Comment:
This should be in the namespace `fluree.db.connection-test` and located at `test/fluree/db/connection_test.cljc` for maximum compatibility with clojure testing libraries & runners. This is because the code its testing doesn't have that `general` component, so this won't be matched up with it.